setup.cfg

```diff
@@ -119,6 +122,7 @@ docs =
   sphinx-design
   sphinx-gallery>=0.13.0 # First to allow us to use a different backend while scraping the images
   sphinxext-opengraph
+  %(spice)s  # because importing sunpy.coordinates.spice raises an error if spiceypy is not installed
```

Comment:
Do we not want this included in `[all]`?

Reply:
It's worth discussing (a la https://github.com/sunpy/sunpy/issues/7066#issuecomment-1591721444).  `spiceypy` is akin to other optional coordinates dependencies (`astroquery`/`jplephem`) that are relevant to only a small subset of users, and those dependencies also aren't included in `[all]`.